docs/source/customizations/custom-binary.mdx

```diff
@@ -22,6 +22,7 @@ To compile the Apollo Router, you need to have the following installed:
 <!-- renovate-automation: rustc version -->
 * [Rust 1.71.1 or later](https://www.rust-lang.org/tools/install)
 * [Node.js 16.9.1 or later](https://nodejs.org/en/download/)
+* [CMake 3.5.1 or later](https://cmake.org/download/)
```

Comment:
@o0Ignition0o Should we retroactively add something to CHANGELOG saying that as of v1.29.0, building Router from source requires CMake to be installed?